bin/index.js

```diff
@@ -213,8 +229,18 @@ module.exports = function(Aquifer, AquiferGitConfig) {
 
       // Commit changes.
       .then(function () {
+        var signature;
+
         Aquifer.console.log('Commit changes...', 'status');
-        var signature = git.Signature['default'](repo);
+
+        if (options.name && options.email) {
```

Comment:
Missing option properties always exist so hasOwnProperty doesn't work. This is because of https://github.com/aquifer/aquifer/blob/dev/lib/extension.command.js#L152, where if the option wasn't specified it gets set to null. Then after I merge the command options, config options, and default options together they are undefined. So should I continue just checking for a falsy value or should I use `typeof property === 'undefined'`?